ansible/roles/hyrax/tasks/compel.yml

```diff
@@ -0,0 +1,44 @@
+---
+- block:
+  - name: load workflow
+    command: ./bin/rails hyrax:workflow:load
```

Comment:
This Rake task is executed for all Hyrax applications as part of the `hyrax/tasks/main.yml` play sequence in the `hyrax` role.  It is superfluous here.